examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -80,10 +80,18 @@ const bool kInvertTxOutput = false;
 // Note: GPIO 16 won't work on the ESP8266 as it does not have interrupts.
 const int8_t kDefaultIrRx = 21;  // <=- CHANGE_ME (optional)
 
+// Default GPIO the RF Receiver is connected
+const int8_t kDefaultRfRx = 12;  // <=- CHANGE_ME (optional)
+
 // Enable/disable receiving/decoding IR messages entirely.
 // Note: IR_RX costs about 40k+ of program memory.
 #define IR_RX true
 
+// Enable/disable receiving/decoding RF messages entirely.
+// Receive/decoding RF signals based on RCSwitch library, at: https://github.com/sui77/rc-switch
+// Note: for enable is necessary a receiver/transmitter 433MHz module.
+#define RF_RX true
+
```

Comment:
is this being repeated from above?

Reply:
never mind, I miss read it, I thought there was two `RF_RX`, but the other is `IR_RX` xD (sorry about that)